.travis.yml

```diff
@@ -15,6 +15,10 @@ cache:
 go:
   - 1.5.1
 
+sudo: required
+services:
+- docker
+
```

Comment:
* `language` should be `generic`; no longer go (since all the go tools are in a container).
* Do not need to set `GO15VENDOREXPERIMENT=1`.  This is already set inside the container.
* Do not need to specify go version.
* All install steps can go away.  Those tools all exist already within the container.  You can replace with a _single_ install step: `make bootstrap`
* `script` should be just `make build test`.  This will run a containerized build for Linux/64 within the container and the tests will be executed within the container, against the code _and_ the binary.
* Everything else looks correct.